billsplit/src/pages/CalculatorPage.js

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from "react";
+import {calculateLogisticsTotal} from '../utils/calculateTotalAmout'
+import Invoice from '../components/Invoice'
+const CalculatorPage = () => {
+  const [values, setValues] = useState({
+    eventTitle: "",
+    amountTotal: "",
+    numberOfPartyMembers: "",
+    taxPercent: "",
+    tipPercent: ""
+  });
+
+  const [total, setTotal] = useState('')
+
+  const [invoice, showInvoice] = useState(false)
+
+
+  const handleInputChanges = e => {
+    const { name, value } = e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  const handleSubmit = e => {
+      e.preventDefault()
+      let total = (calculateLogisticsTotal(values))
+    setTotal(total)
+      showInvoice(true)
+  }
+
+  
+  const showInvoiceOrForm = () => {
+      if (invoice) {
+          return <Invoice eventTitle={values.eventTitle} numberOfPartyMembers={values.numberOfPartyMembers} amountTotal={values.amountTotal} taxPercent={values.taxPercent} tipPercent={values.tipPercent} totalPerPerson={total} />
```

Comment:
It would be easier to follow your props if you split the props being passed into this component on different lines, i.e.

```
<Invoice 
     eventTitle={values.eventTitle} 
     numberOfPartyMembers={values.numberOfPartyMembers} 
     amountTotal={values.amountTotal} 
     taxPercent={values.taxPercent} 
     tipPercent={values.tipPercent} 
     totalPerPerson={total} 
/> 
```